code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -547,6 +640,10 @@
 			cyborg_base_icon = "tophat"
 			special_light_key = null
 			hat_offset = INFINITY //He is already wearing a hat
+		if("Heavy")
+			cyborg_base_icon = "heavyserv"
+			special_light_key = "heavyserv"
+

```

Comment:
Surround these changes with
`// austation begin -- <reason for change>` on a line before the start of the modified code
and
`// austation end` on a line following the changes

Reply:
lower case, space after the //, then after the austation start, a -- then the reason for the edit, also needs have have begin after the // austation
same as above, remove the reason, add a space after //, lower case, make it austation end

Reply:
you misspelled austation, and add the `begin` after `// austation`